stake-pool/program/tests/helpers/mod.rs

```diff
@@ -2379,12 +2379,14 @@ pub fn add_validator_stake_account(
         credits_observed: 0,
     };
 
+    let mut data = bincode::serialize::<stake::state::StakeState>(
+        &stake::state::StakeState::Stake(meta, stake),
+    )
+    .unwrap();
+    data.extend_from_slice(&[0, 0, 0, 0]);
```

Comment:
i havent run the code to see what happens but shouldnt this have a check that the length is 196 before extending? otherwise when we move to 1.17 we would extend to 204

Reply:
good point, I'll instead allocate a bigger vec and then write into it